stock_picking_send_by_mail/models/stock_picking.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 - Sandra Figueroa Varela
+# Copyright 2017 Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import _, api, models
+
+
+class StockPicking(models.Model):
+    _name = "stock.picking"
+    _inherit = ['stock.picking', 'mail.thread']
```

Comment:
Why do you inherit from `mail.thread`? It's not needed for sending mails (you can send it as mail.message). Anyway, it can be convenient to have a thread for following the conversation, but in that case, you should rename the module to `stock_picking_mail` for being appropriated for the new features and add the chatter to the form.